docs/guide/content/development_guide/github_actions.md

```diff
@@ -1,17 +1,18 @@
 # GitHub Actions
+This project uses several GitHub Actions workflows to automate testing, releases, and documentation.
+Access to these workflows varies based on user permissions.
 
-This project uses several GitHub Actions workflows to automate testing, releases, and documentation. Below is a summary of each workflow and when it runs.
+For the actual definition of the workflows refer to the [repository source](https://github.com/IQSS/ondemand-loop/tree/main/.github/workflows).
 
-## Workflow Summaries
+### Workflow Summaries
 
 - **test.yml** – Runs the full test suite via `make test` on pull requests and pushes to the `main` branch. It also generates SimpleCov coverage badges when changes are pushed to `main`.
 - **guide.yml** – Builds this guide using `make guide` and deploys it to GitHub Pages whenever documentation under `docs/guide` changes on `main`.
-- **build.yml** – Manually triggered workflow used to create a production build from a given tag and push the result to the `iqss_production` branch (no deployment occurs).
```

Comment:
why we are no longer documenting this? the action is still in the project

Reply:
This build.yml is only there to create releases manually in case the issue driven way fails. It will be removed once we get into a routine with releases